src/Symfony/Component/HttpKernel/Controller/ControllerResolver.php

```diff
@@ -40,6 +47,14 @@ public function __construct(LoggerInterface $logger = null)
     }
 
     /**
+     * @internal
+     */
+    public function setArgumentResolverManager(ArgumentResolverManager $argumentResolverManager)
```

Comment:
@sstok the problem is that the ControllerResolver is instantiated *outside* the kernel and injected through the controller. so there would be no way in setting the argument resolver then.